src/test/AludelFactory.t.sol

```diff
@@ -17,7 +17,7 @@ import "forge-std/src/console2.sol";
 
 contract AludelFactoryTest is DSTest {
     AludelFactory private factory;
-    Hevm private cheats;
+    Vm private cheats;
```

Comment:
If this contract extends Test instead of DSTest, then this is automatically configured